src/main/java/frc/team4909/robot/Robot.java

```diff
@@ -121,33 +131,34 @@ public void robotInit() {
         RobotConstants.manipulatorGamepadSensitivity // Gamepad sensitivity
     );
 
-    climberGamepad = new BionicF310(3, // Port
+    climberGamepad = new BionicF310(RobotMap.climberGamepadPort, // Port
         RobotConstants.manipulatorGamepadDeadzone, // Deadzone
         RobotConstants.manipulatorGamepadSensitivity // Gamepad sensitivity
     );
 
 
-    climbStick = new FlightStick(2);
-
-    /* Drivetrain */
-
     /* Intake */
-    manipulatorGamepad.buttonHeld(BionicF310.RT, 0.2, new CargoIntakeIn());
-    manipulatorGamepad.buttonHeld(BionicF310.RB, new CargoIntakeOut());
-    manipulatorGamepad.buttonHeld(BionicF310.LT, 0.2, new HatchPanelIntakeOpen());
+    manipulatorGamepad.buttonHeld(BionicF310.RT, 0.2, new CargoIntakeOut());
+    manipulatorGamepad.buttonHeld(BionicF310.LT, 0.2, new CargoIntakeIn());
+    manipulatorGamepad.buttonHeld(BionicF310.RB, new HatchPanelIntakeOpen());
```

Comment:
This (and the two subsequent) `buttonHeld` calls have no float value as a second parameter, whereas the two above it do. Is there a missing parameter? Default value? I assume that parameter is the time it must be held before invoking the command, yes? If this last one is using a default, I recommend specifying the value here even though there is an option to use a default. It's much clearer to the reader if all are specified.

Reply:
The triggers of the gamepad require the "0.2", which is a threshold. The bumpers do not have a threshold part of the constructor.

Reply:
it's a threshold, for the triggers only. it takes an axis and considers it like a button IF it exceeds that threshold

Reply:
there is no default value otherwise